.goreleaser.yml

```diff
@@ -11,9 +11,9 @@ builds:
     binary: compass
     flags: [-a]
     ldflags:
-      - -X github.com/odpf/compass/cli.Version={{.Tag}}
-      - -X github.com/odpf/compass/cli.BuildCommit={{.FullCommit}}
-      - -X github.com/odpf/compass/cli.BuildDate={{.Date}}
+      - -X github.com/goto/compass/cmd.Version={{.Tag}}
```

Comment:
This should still be cli right?

Reply:
nice catch, will resolve this, there's an issue with the cherry-picking